### ISSUE TITLE
Entity-filter-badge does not work (0.110.4)

### ISSUE THREAD

**Issue by tdejneka** [home-assistant/frontend#6077]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem
Entity-filter-badge fails to display badges when filter conditions are met.  This problem was previously reported in Issue [6047](https://github.com/home-assistant/frontend/pull/6050) and corrected in PR [6050](https://github.com/home-assistant/frontend/pull/6050). However, I just tested it in 0.110.4 and (after clearing the browser cache many times) still fails to work correctly (it worked in versions prior to 0.110.0).

## Expected behavior
Should reveal the badge when its filter conditions are met.

## Steps to reproduce
See Issue [6047](https://github.com/home-assistant/frontend/pull/6050) and/or sample config below.

## Environment
Browser: Firefox 76.0.1 on Linux

arch | x86_64
-- | --
chassis | laptop
dev | false
docker | true
hassio | true
host_os | Ubuntu 18.04.4 LTS
installation_type | Home Assistant Supervised
os_name | Linux
os_version | 5.3.0-53-generic
python_version | 3.7.7
supervisor | 225
version | 0.110.4
virtualenv | false

- Home Assistant release with the issue: 0.110.4
- Last working Home Assistant release (if known): 0.109.x
- Browser and browser version: Firefox 76.0.1 (Linux)
- Operating system: Ubuntu 18.04.4

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml
views:
  - badges:
      - entity: sensor.indoor_temperature
      - entity: sensor.indoor_humidity
      - entity: sensor.outdoor_temperature
      - entities:
          - sensor.all_doors
          - sensor.all_lights
          - sensor.all_appliances
        state_filter:
          - operator: '!='
            value: '0'
        type: entity-filter
```

## Javascript errors shown in your browser console/inspector
none.

```txt

```

## Additional information
If I open the Raw Configuration Editor then close it to return to the UI, the badges are displayed correctly. If I navigate away and return, or simply press F5 to refresh the page, the badges disappear.

After existing Raw Configuration Editor; badges are displayed properly (because their values are non-zero).
![Screenshot from 2020-05-30 13-01-56](https://user-images.githubusercontent.com/44732909/83334875-71117980-a277-11ea-83fd-a81f54b41a8b.png)

After refreshing the page; badges disappear (despite their values remaining non-zero).
![Screenshot from 2020-05-30 13-02-16](https://user-images.githubusercontent.com/44732909/83334850-48897f80-a277-11ea-9587-3b4ce4ac82af.png)

**Comment by tdejneka** [comment]
Just realized that my anticipation for this correction blinded me to the fact it was merged to dev branch but never made it to 0.110.4 release!

**Comment by bramkragten** [comment]
My bad, forgot to tag it for 0.104!
